generator/generator.py

```diff
@@ -0,0 +1,47 @@
+import time
+import random
+from datetime import datetime
+import json
+from locust import HttpUser, task, between, events
+
+class UserBehavior(HttpUser):
+    wait_time = between(1, 3)
+    
+    def on_start(self):
+        pass
+        
+    @task(2)
+    def sum_test(self):
+        x = random.randint(1, 100)
+        y = random.randint(1, 100)
+        payload = {"x": x, "y": y}
+        
+        with self.client.post("/sum", json=payload, catch_response=True) as response:
+            try:
+                expected = str(x + y)
+                if response.text == expected:
+                    response.success()
+                else:
+                    response.failure(f"Got {response.text}, expected {expected}")
+            except Exception as e:
+                response.failure(f"Exception: {str(e)}")
+    
+    @task(1)
+    def matmul_test(self):
+        x = [[random.randint(1, 10) for _ in range(3)] for _ in range(3)]
+        y = [[random.randint(1, 10) for _ in range(3)] for _ in range(3)]
+        payload = {"x": x, "y": y}
```

Comment:
the api generates the matrix, x and y makes up the size of the matrices

matrix 1 will be m=x, n=y while matrix 2 will be m=y, n=x

therefore:

```suggestion
        payload = {
            "x":random.randint(1, 10),
            "y":random.randint(1, 10)
        }
```